bin/check-bundle-size.js

```diff
@@ -52,7 +53,7 @@ log('File sizes look OK.');
 
 function isTooBig({ path, size }) {
   // Special cases:
-  const simpleName = basename(path).replace(/-\w{8}\./, '.');
+  const simpleName = basename(path).replace(/-.{8}\./, '.');
```

Comment:
\w doesn't include `-` which can be part of the generated file. See `[check-bundle-size]  444452 dist/assets/index-DMq9Y-6m.js` in https://github.com/getodk/central-frontend/actions/runs/17078821495/job/48427136906